internal/ministry/client.go

```
@@ -4,216 +4,384 @@ import (
+		c.setStdHeaders(req)
+
+		res, err := c.HTTPClient.Do(req)
+		if err != nil {
+			// Never retry caller cancellations or deadline overruns.
+			if errors.Is(err, context.Canceled) || errors.Is(err, context.DeadlineExceeded) {
+				return err
+			}
+			lastErr = err
+		} else {
+			if res.StatusCode == http.StatusOK {
+				defer res.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `res.Body.Close` is not checked

(errcheck)